app/views/rules/index_search.html.erb

```diff
@@ -66,3 +68,30 @@ $(document).ready(function(){
       });
 });
 </script>
+
```

Comment:
Can you include this javascript in the assets pipeline?
Me and Bruno are also struggling with the pipeline, but it is a more elegant way, besides making the code more easily maintainable.
https://web-crunch.com/posts/understanding-asset-pipeline-ruby-on-rails

Reply:
Certainly. I'll do this today.